gitea/repo_collaborator.go

```diff
@@ -11,26 +11,21 @@ import (
 )
 
 // ListCollaborators list a repository's collaborators
-func (c *Client) ListCollaborators(user, repo string) ([]*User, error) {
-	collaborators := make([]*User, 0, 10)
+func (c *Client) ListCollaborators(user, repo string) ([]*Collaborator, error) {
+	collaborators := make([]*Collaborator, 0, 10)
 	err := c.getParsedResponse("GET",
 		fmt.Sprintf("/repos/%s/%s/collaborators", user, repo),
 		nil, nil, &collaborators)
 	return collaborators, err
 }
 
 // IsCollaborator check if a user is a collaborator of a repository
-func (c *Client) IsCollaborator(user, repo, collaborator string) (bool, error) {
-	status, err := c.getStatusCode("GET",
+func (c *Client) IsCollaborator(user, repo, collaborator string) (*Permission, error) {
```

Comment:
This is breaking change and will break current SDK

Reply:
Hmm... This was to go with the associated pull-request. Would it better to create a new API end-point to retrieve permissions?

Similarly does the change with reporting the permission in the /repos/{owner}/{repoName}/collaborators break the SDK?

Reply:
For getting permissions at least new function has to be created even if it would be using same endpoint.

As for type change that is minor breakage as you will still get struct with same properties

Reply:
Ah ok. I can do that.

Reply:
Have restored the signature.